doc/fault-injection.md

```diff
@@ -0,0 +1,152 @@
+# Troubleshooting with Istio: Simulating slow responses from microservices
+
+![Fault Injection](fault-injection.png)
+
+Slow or unresponsive microservices can reveal defects in other system components.  Unhandled timeout
+exceptions can cascade and affect user experience.  Developers get frustrated trying to diagnose, correct,
+and verify fixes to performance problems on their speedy and responsive test systems.
+
+In this tutorial we will show advanced features of the Istio command line to inject performance problems into
```

Comment:
s/performance problems/delays/
In this tutorial, we will show how to use Istio to inject delays into API calls between two microservices.